setup.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Setup tool for artman."""
+
+import setuptools
+
+from pip.req import parse_requirements
+
+requirements = [
+    'oslo.utils>=3.4.0',
+    'pyyaml>=3.11',
+    'taskflow>=1.25.0',
+    'yapf>=0.6.2'
```

Comment:
Just realize that PR16 (https://github.com/googleapis/artman/pull/16/files) hasn't been merged. Can you add the three extra requirements listed in that PR into this one?